DistributionSystemApi/MailLibrary/DistributionSystemApi.MailLibrary.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
also please add an extension which registers all important services in service collection